data/api/server-server/query.yaml

```diff
@@ -170,6 +170,20 @@ paths:
                     "displayname": "John Doe",
                     "avatar_url": "mxc://matrix.org/MyC00lAvatar"
                   }
+        "403":
+          x-addedInMatrixVersion: "1.2"
+          description: The server is unwilling to disclose whether the user exists and/or
+            has a display name.
+          content:
+            application/json:
+              schema:
+                $ref: ../client-server/definitions/errors/error.yaml
+              examples:
+                response:
+                  value: {
+                    "errcode": "M_FORBIDDEN",
+                    "error": "Profile lookup over federation is disabled on this homeserver"
+                  }
```

Comment:
actually: given MSC3550 didn't mention the federation API at all, making this change without an MSC seems like a bit of a stretch.

Reply:
(and while we're at it maybe we should discuss the additional client endpoints too, since they weren't *technically* covered by the MSC or the original spec PR)

Reply:
> given MSC3550 didn't mention the federation API at all

It mentions that the reason the MSC is being written is for requests over federation, but then doesn't mention the federation endpoints at all.

I'm a bit torn on it.

Reply:
I am (not eager but definitely) willing to put up a short MSC for this. I would like to ensure that it's time well spent though. So I'll hold off of doing anything until you've found an agreement on whether an MSC is needed or not.

Reply:
We discussed it briefly at the time of richvdh's comment, and I think the thread here captures that discussion well. It feels like it needs an MSC because it's relatively unclear what the original MSC's intended scope was. 

As a formality, my vote is +1.0 to needing an MSC because the intention and history are different stories.

Reply:
Ok, makes sense. I'll try and put something together next week.

Reply:
This PR is now blocked on https://github.com/matrix-org/matrix-spec-proposals/pull/4170

Reply:
This PR has now been updated as per the accepted MSC.